### PR TITLE
Feature/caes 682  adm panel users roles

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -55,6 +55,8 @@ MAILER_PASSWORD=
 #DELIVERY_ADDRESS=
 #DB_VERSION=9.6
 #WEB_CLIENT_URL=
+#### When registering, a user with such an email will be assigned as admin
+#CHOOSEN_ADMIN_EMAIL=test4@test.ty
```
Comment (beshkenadze) [inline .env.dist]: SUPREME_ADMIN_EMAIL or DOMAIN_ADMIN_EMAIL are preferred names.

Comment (mg-4xxi) [reply .env.dist]: changed

File: public/static/js/form_login.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                'B': response.data.publicEphemeralValue,
+                'seed': response.data.seed,
+            });
+        })
+        .catch(function (error) {
+            console.log(error);
+        });
+
+    return false;
+}
```
Comment (beshkenadze) [inline public/static/js/form_login.js]: please add a new line into this file.

File: public/static/js/form_login.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+            generateSession(data);
+        })
+        .catch(function (error) {
+            console.log(error);
+        });
+}
+
+document.getElementById('login').onsubmit = function () {
+    console.log(this);
```
Comment (beshkenadze) [inline public/static/js/form_login.js]: please remove all console.log methods.

File: public/static/js/form_login.js (hunk 2 above)
Comment (mg-4xxi) [reply public/static/js/form_login.js]: done

File: public/static/js/form_login.js (hunk 3 above)
Comment (mg-4xxi) [reply public/static/js/form_login.js]: done

**Pull request merged**